.gitignore

```diff
@@ -27,6 +27,6 @@ logs/
 *.received.json
 
 # Contract Tests
-pact/
+*/src/test/pact/
```

Comment:
I know I literally gave you this line, but could you pls add a leading `/` to anchor it to the root of the repository?

Reply:
done